src/ui/CategoryFacet/CategoryFacetSearch.ts

```diff
@@ -35,6 +35,8 @@ export class CategoryFacetSearch implements IFacetSearch {
     return CategoryFacet.ID;
   }
 
+  public set currentFacetSearchElementId(id: string) {}
```

Comment:
will add it on a future PR?

Reply:
This was added to implement the `IFacetSearch` interface, but most likely this function will need to do something in a future PR.